evalml/pipelines/time_series_pipeline_base.py

```diff
@@ -183,11 +183,19 @@ def predict(self, X, objective=None, X_train=None, y_train=None):
             y_train (pd.Series or None): Training labels.
 
         Raises:
-            ValueError: If final component is not an Estimator.
+            ValueError: If X_train and/or y_train are None or if final component is not an Estimator.
 
         Returns:
             Predictions.
         """
+        if X_train is None:
+            raise ValueError(
+                "Make sure to have a non None value for X_train when calling time series' predict and that in the predict method you set X_train = X_train"
```

Comment:
I don't think you need the "and that in the predict method you set X_train = X_train" here - mirroring the y_train error message is perfect!